plugin/pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -24,19 +24,34 @@ const (
 	memoryResoureceName  string = "Memory"
 )
 
+var (
+	// The predicateName tries to be consistent as the predicate name used in DefaultAlgorithmProvider defined in
+	// defaults.go (which tend to be stable for backward compatibility)
```

Comment:
I think you are missing MaxEBSVolumeCount and MaxGCEPDVolumeCount, which were added recently (see defaultPredicates())